tools/boards.txt.py

```diff
@@ -1075,7 +1077,7 @@
 
     'flash_erase_menu': collections.OrderedDict([
         ( '.menu.wipe.none', 'Only Sketch' ),
-        ( '.menu.wipe.none.upload.erase_cmd', 'version' ),
+        ( '.menu.wipe.none.upload.erase_cmd', 'flash_id' ),
```

Comment:
I am concerned about this change. 
`flash_id` asks for a connection to the esp, while `version` is a no-op which is what we need in this case.

Reply:
Take a board without reset circuit and version as first default command as you want :
- "Only sketch" works only with the "no dtr" reset method
- "Flash erase" works only with the "no dtr, no sync" reset method or second programming step will fail.

Do we really want the user change the reset method every time the erasing flash option is changed ?

With flash_id, both "Only sketch" and "Flash erase" works with "no dtr, no sync" reset method.

As described, this is a workaround needed since the esptool do the erasing and programming in two consecutive commands.  May need also the "no dtr, no sync" reset method for all "ck" boards.